src/test/java/ca/bc/gov/backendstartapi/repository/FundingSourceRepositoryTest.java

```diff
@@ -18,8 +19,19 @@
 @AutoConfigureTestDatabase(replace = Replace.NONE)
 class FundingSourceRepositoryTest {
 
-  @Autowired
-  private FundingSourceRepository fundingSourceRepository;
+  @Autowired private FundingSourceRepository fundingSourceRepository;
+
+  private boolean isValid(FundingSource fundingSource) {
+    LocalDate today = LocalDate.now();
+
+    // Effective date - Should be before today
+    if (!fundingSource.getEffectiveDate().isBefore(today)) {
+      return false;
+    }
```

Comment:
A funding source is valid if `effectiveDate` ≤ today < `expiryDate`, so the effective date should be _today or before_.